github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
Oh! Here be jdbc dragons - perhaps we can improve it for users on JRuby?

Compare [this change](https://github.com/earaujoassis/sequel-seed/commit/9a470a97949efe41d505cc21f3bcbba8c816070d), to a completely different project, for inspiration.